cost/azure/hybrid_use_benefit/azure_hybrid_use_benefit.pt

```diff
@@ -160,40 +169,102 @@ EOS
 end
 
 script "js_filter_resources", type: "javascript" do
-  parameters "azure_instances","exclusion_tag"
+  parameters "azure_instances","exclusion_tag", "pricing"
```

Comment:
parameter variable names should be identical to their datasource or parameter counterparts when applicable. See: https://flexera.atlassian.net/wiki/spaces/FP/pages/2184642622/Policy+Best+Practices#Scripts